### DISCUSSION TITLE
Looking to have a call with the maintainers.

### DISCUSSION THREAD

**Discussion by MikeCodesDotNET** [AvaloniaCommunity/Prism.Avalonia#63] [General]
Hey all, 

I'm looking to have a conversation with the maintainers. Who would be best to speak with?

**Comment by DamianSuess** [comment]
Hi @MikeCodesDotNET, what's on your mind?
If you'd like to reach out in private, feel free via [LinkedIn](https://www.linkedin.com/in/damiansuess/)

**Comment by DamianSuess** [comment]
@MikeCodesDotNET, hope all is well. I'm reaching out to let you know that Prism.Avalonia has been updated to support Avalonia's v11.0 GA release. If you don't mind, i'm curious about what's coming down the pipeline in the future so that I can prepare this library for it. Also, if there is anything you'd like to see added/supported by this library, let me know, I'm happy to help.

In the near future, I'm planning on expanding the samples and writing a few articles to showcase the latest v11.0 features.

Take care,

Damian

| Package | Download
|-|-|
| Prism.Avalonia | [![Prism.Avalonia NuGet Badge](https://buildstats.info/nuget/Prism.Avalonia?dWidth=70)](https://www.nuget.org/packages/Prism.Avalonia/8.1.97.11000)
| Prism.DryIoc.Avalonia | [![Prism.DryIoc.Avalonia NuGet Badge](https://buildstats.info/nuget/Prism.DryIoc.Avalonia?dWidth=70)](https://www.nuget.org/packages/Prism.DryIoc.Avalonia/8.1.97.11000)
